frontend/app/services/query-result.js

```diff
@@ -248,6 +249,10 @@ function QueryResultService($resource, $timeout, $q) {
             yValues[name] = value;
             point[type] = value;
           }
+          if (type === 'error_y') {
```

Comment:
Mh, I am a bit confused by the implementation of the whole method. But I believe it needs to be `error_y` because it's already the data structure that is feed to plotly (and the parameter that plotly want as input is called `error_y`). I will double check anyways